index.html

```diff
@@ -114,7 +114,38 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
-	 
+	 <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
Please add a new line between your card and the template. Otherwise it would be hard for the others to identify the code.

Reply:
I made the requested changes. I'll do a follow-up pull request.

Reply:
Unless I don't need to make another pull request. Can you access the changes commit I made?